index.html

```diff
@@ -14,172 +14,200 @@
+                    <li><a href="#"><img src="img/icons/Vector.png" alt=""></a></li>
+                    <li><a href="#"><img src="img/icons/Linkedin_icon.png" alt=""></a></li>
+                    <li><a href="#"><img src="img/icons/Vector-1.png" alt=""></a></li>
+                    <li> <a href="#"><img src="img/icons/Vector-2.png" alt=""></a></li>
+                    <li> <a href="#"><img src="img/icons/Vector3.png" alt=""></a></li>
+                </ul>
+            </div>
         </section>
         <section class="work-cards-container">
             <div class="work-card work-card1">
-                <img src="img/works/work1.png" alt="work1">
-
-                <h2 class="work-title">Tonik</h2>
-                <div class="work-info">
-                    <p class="first">CANOPY</p>
-                    <p class="second">• Back End Dev</p>
-                    <p class="second">• 2015</p>
-                </div>
-                <div class="word-desc">
-                    <p>A daily selection of privately personalized reads; no accounts or sign-ups required.</p>
+                <div class="card-img">
+                    <img src="img/works/work1.png" alt="work1">
                 </div>
-                <div class="work-lang">
-                    <ul>
-                        <li>
-                            <p class="lang-text">html</p>
-                        </li>
-                        <li>
-                            <p class="lang-text">css</p>
-                        </li>
-                        <li>
-                            <p class="lang-text">javascript</p>
-                        </li>
-                    </ul>
-                </div>
-                <button type="button" class="work-btn btn">
+                <div class="card-details">
+                    <h2 class="work-title">Tonik</h2>
+                    <div class="work-info">
+                        <p class="first">CANOPY</p>
+                        <p class="second">• Back End Dev</p>
+                        <p class="second">• 2015</p>
+                    </div>
+                    <div class="word-desc">
+                        <p>A daily selection of privately personalized reads; no accounts or sign-ups required.</p>
+                    </div>
+                    <div class="work-lang">
+                        <ul>
+                            <li>
+                                <p class="lang-text">html</p>
+                            </li>
+                            <li>
+                                <p class="lang-text">css</p>
+                            </li>
+                            <li>
+                                <p class="lang-text">javascript</p>
+                            </li>
+                        </ul>
+                    </div>
+                    <button type="button" class="work-btn btn">
                         See Project
                     </button>
+                </div>
```

Comment:
- [ ] The image of the project is overflowing and it should be like the one in the Figma design in order to match Design please try to fix  that 
- Figma
![image](https://user-images.githubusercontent.com/61790587/171681189-26dceebc-711f-4ccd-84a0-54341d69babb.png)
- Yours 
![image](https://user-images.githubusercontent.com/61790587/171681220-0d74f39e-ceb4-4748-a5fd-d51d8962ef68.png)